pkgs/dart_services/lib/src/flutter_genui.dart

```diff
@@ -38,11 +38,12 @@ class _GenuiEnv {
   /// Returns the generated Flutter code.
   ///
   /// If not enabled or fails, logs error and returns null.
-  Future<String?> request({required String prompt}) async {
+  Future<String> request({required String prompt}) async {
     final uri = apiUrl;
     if (uri == null) {
-      _logger.warning('Genui features at $name are disabled');
-      return null;
+      throw Exception(
```

Comment:
Maybe subclass `Exception` to something that is more relevant to this part of the code path? `GenUiException` or something similar. It seems that `name` could be a common field, it's used to compose the message in both cases?

Reply:
It is just Exception across the DartPad. And I do not see value in introducing additional types for now. Will stick with YAGNI